.pre-commit-hooks.yaml

```diff
@@ -328,3 +328,13 @@
     openapi
   language: docker_image
   files: '^apidoc/.+/openapi\.yaml$'
+- id: prettier-xml
+  name: prettier-xml
+  additional_dependencies:
+    - 'prettier@2.7.1'
+    - '@prettier/plugin-xml@2.2.0'
+  entry: prettier --write --list-different --ignore-unknown
+  language: node
+  files: \.xml$
+  stages: [commit, push, manual]
```

Comment:
After we verify that it works in the first place: can we remove this and check if it works the same?

Reply:
removed, works